src/main/java/webserver/servlet/UserLoginController.java

```diff
@@ -24,14 +34,13 @@ public void doPost(HttpRequest httpRequest, HttpResponse httpResponse) {
             .orElseThrow(() -> new IllegalStateException(NOT_FOUND_MEMBER));
 
         if (!userFound.fitPassword(password)) {
-            httpResponse.addHeader(HttpHeader.SET_COOKIE, KEY_LOGINED + "=" + false + "; " + "Path=/");
+            httpResponse.addCookie(Cookie.notLoginedWithPath("/"));
             httpResponse.sendRedirect("/user/login_failed.html");
             return;
         }
 
-        HttpCustomSession session = Sessions.INSTANCE.create();
-        session.setAttribute(KEY_LOGINED, Boolean.TRUE.toString());
-        httpResponse.addHeader(HttpHeader.SET_COOKIE, HttpHeader.JSESSIONID + "=" + session.getId());
+        HttpSession session = httpRequest.getSession();
+        httpResponse.addCookie(session.getLoginedCookie());
```

Comment:
```suggestion
        HttpSession session = httpRequest.getSession();
        session.add(key, value);
```
일반적으로 Session 을 사용하는 코드는 위와 같이 구현하는게 깔끔하다고 생각합니다. 
`httpResponse.addCookie(session.getLoginedCookie());` 와 같이 Response 에 SesseionId를 넣어주는것이 필요는 하지만, 개발자가 직접 다루기엔 휴면에러가 발생할 가능성이 너무 큰것 같아요. 

HttpRequest에서 Session을 가져올때  세션이 있다면, 기존 세션을 가져와야 하고, 없다면 새로 생성해주어야 합니다. 여기까지 구현은 잘해주셨어요 👍 
다음 스탭으로 Response 에 SessionId 를 실어주어야 하는데, Tomcat 에서 Session 구현을 위해 Response 객체를 어떻게 다루었는지 한번 보시면 좋을것 같아 링크 남겨드려요 ㅎㅎ 

https://github.com/apache/tomcat/blob/6916f850b0075513aea68b762d8aa530b3b5350b/java/org/apache/catalina/connector/Request.java#L677-L684

톰캣에서는 Request 객체가 Response 객체를 가지는 구조로 처리하였는데, 참고만 해주시면 될것 같습니다. ㅎㅎ 
저는 미션을 구현할때 ThreadLocal을 활용하여 Response 객체를 저장하고, Session 초기화시 응답 헤더에 추가해주는 방법으로 구현했던것 같아요 😄

Reply:
오.. 읽어보고 읽은 내용이랑 동철님 방법이랑 참고해서 생각을 좀 해보고 적용을 해겠습니다!!